media-tv/tvbrowser-bin/tvbrowser-bin-4.0.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2019 Gentoo Foundation
```

Comment:
This copyright header is old, use repoman to commit so it fixes it for you automatically ;)